modules/avatar/avatar_test.go

```diff
@@ -13,12 +13,17 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func Test_RandomImage(t *testing.T) {
-	_, err := RandomImage([]byte("gogs@local"))
+func Test_RandomImageSize(t *testing.T) {
+	_, err := RandomImageSize(0, []byte("gitea@local"))
+	assert.Error(t, err)
```

Comment:
which error ?

Reply:
That an image of size 0 is invalid. It returns a chinese error currently:

```
identicon.New: 参数 size 的值(0)不能小于 16
```